### ISSUE TITLE
fflush(stdout); causes Segfault when using BitCode Parser

### ISSUE THREAD

**Issue by pointhi** [graalvm/sulong#474]
This bug happens only when using the bitcode parser

```c
#include<stdio.h>

int main(void)
{
    printf(".");
    fflush(stdout);

    return 0;
}
```

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f37fab9d0f8, pid=22283, tid=0x00007f37fb2d1700
#
# JRE version: OpenJDK Runtime Environment (8.0_91-b14) (build 1.8.0_91-8u91-b14-0ubuntu4~14.04-b14)
# Java VM: OpenJDK 64-Bit Server VM (25.71-b01-internal-jvmci-0.20 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libc.so.6+0x670f8]  fflush+0xb8
#
# Core dump written. Default location: /home/thomas/JKU/sulong-dev/sulong/core or core.22283
#
# An error report file with more information is saved as:
# /home/thomas/JKU/sulong-dev/sulong/hs_err_pid22283.log
Compiled method (JVMCI)    5447 4836       4       com.oracle.graal.truffle.hotspot.nfi.NativeCallStubGraphBuilder$NativeCallStub::libCall (6 bytes)
 total in heap  [0x00007f37e5b836d0,0x00007f37e5b839c0] = 752
 relocation     [0x00007f37e5b83808,0x00007f37e5b83818] = 16
 main code      [0x00007f37e5b83820,0x00007f37e5b83928] = 264
 oops           [0x00007f37e5b83928,0x00007f37e5b83930] = 8
 metadata       [0x00007f37e5b83930,0x00007f37e5b83940] = 16
 scopes data    [0x00007f37e5b83940,0x00007f37e5b83958] = 24
 scopes pcs     [0x00007f37e5b83958,0x00007f37e5b839b8] = 96
 dependencies   [0x00007f37e5b839b8,0x00007f37e5b839c0] = 8
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
#
```

**Comment by jakre** [comment]
This bug is caused by #467. Memory for stdout is already allocated externally when the program is run, but the Bitcode Parser allocates it again. The code of the Textparser relies on a certain behaviour of apparently all compilers we use that simply is not present in Bitcode, which is why this works with the Textparser but crashes for the Binary Parser.

**Comment by mrigger** [comment]
Solved by #476. Can you please confirm @pointhi?
